.codeclimate.yml

```diff
@@ -1,4 +1,5 @@
 exclude_paths:
 - vendor/**/*
 - ZeroBin/**/*
-- tests/**/* 
```

Comment:
Please don't guess. You can't ask for something to be merged if you don't know what the code does.